src/tests/suite/daos_rebuild.c

```diff
@@ -1288,18 +1299,15 @@ rebuild_kill_rank_during_rebuild(void **state)
 				      0, NULL);
 	}
 
-	arg->rebuild_cb = rebuild_destroy_pool_cb;
+	arg->rebuild_cb = rebuild_kill_rank_cb;
```

Comment:
in the cb it kills ranks_to_kill[1], but ranks_to_kill[1] seems not inited in this test?

BTW, what's the purpose of set no_stat_clear flag?